src/components/drawer/drawer.style.js

```diff
@@ -0,0 +1,111 @@
+import styled, { css } from 'styled-components';
+import IconButton from '../icon-button';
+
+const StyledDrawerChildren = styled.div`
+  flex: 1;
+  margin-left: 1px;
+`;
+
+const StyledDrawerSidebar = styled.div`
+  margin-top: 60px;
+`;
+
+const StyledDrawerContent = styled.div`
+  min-width: 40px;
+  width: 40px;
+  min-height: 40px;
+  height: auto;
+  position: relative;
+  overflow: hidden;
+  border-right: 1px solid #d9e0e4;
+
+  @keyframes sidebar-visible {
+    0% {
+      opacity: 0;
+    }
+    50% {
+      opacity: 0;
+    }
+    100% {
+      opacity: 1;
+    }
+  }
+
+  @keyframes sidebar-hidden {
+    0% {
+      opacity: 1;
+    }
+    100% {
+      display: none;
+      opacity: 0;
+    }
+  }
+
+  ${StyledDrawerSidebar} {
+    display: none;
+    opacity 0;
```

Comment:
These styles should be declared in `StyledDrawerSidebar`, avoid cascading if it's not beneficial in any way

Reply:
Also, there's a missing colon `:` in `opacity 0;` here

Reply:
fixed